quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Main.java

```diff
@@ -67,6 +67,7 @@
                 Import.class,
                 ShowConfig.class,
                 Tools.class,
+                Service.class,
```

Comment:
Unless there's a reason for users to still be able to call the service command when isWindows returns false, we can also consider removing or adding conditionally in Picocli.createCommandLine

```
if (!windows) {
  spec.subcommands().remove("service");
}
```

or not even include the Service class in the Main annotation, then add it conditionally. This is inline with a small optimization that may be done in the future to only initially add subcommands if their command name appears in the argument list.

Reply:
> Regardless of the approach, HelpCommandDistTest would still require some changes, right? (unless we want to introduce a separate set of templates)

That is correct - we could also consider not even running them on windows.

Reply:
If a system property or environment variable also controlled the presense of the service command the HelpCommandDistTest could selectively turn it off.

Alternatively if it's easy to just pass all command options from one .bat to another, we could have a kc-service.bat script set a  system property or environment variable to then call kc.bat - then normal usage of kc.bat won't expose the service command.